src/components/carousel/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as CarouselDots } from './CarouselDots';
```

Comment:
I think you copied the carourals from the MUi template, but are you currently actually using them?

If not, I would not include them untill we actually use them.